bin/fui

```diff
@@ -8,11 +8,12 @@ program_desc 'Find unused imports in an Objective-C codebase'
 
 flag [:p, :path], desc: 'Path to search', default_value: Dir.pwd
 switch [:v, :verbose], desc: 'Verbose', default_value: false
+switch [:x, :excludeselfxib], desc: 'Exclude self xib from search (e.g. if false, Foo.h will be marked as used if a Foo.xib reference the Foo class)', default_value: false
 
 default_command :find
 
 pre do |global_options, command, options, args|
-  $fui = Fui::Finder.new(global_options[:path])
+  $fui = Fui::Finder.new(global_options[:path], global_options[:excludeselfxib])
```

Comment:
Should refactor `Finder#initialize` to be `(path, options = {})` because `path` is required, but the rest is not, and just pass global_options in without `path`.